### PR TITLE
Improvements for ":noclip" & ":clip" commands.

### PR DESCRIPTION
**Pull request by EgizianoEG**
Why there was another cloned clipper script?! 
Also, there was unnecessary code inside the ":clip" command which is the value of the Enabled Boolean value...

### CODE REVIEW

File: MainModule/Server/Commands/Moderators.lua [hunk 1]
```diff
@@ -2503,17 +2503,18 @@ return function(Vargs, env)
...
 			AdminLevel = "Moderators";
 			Function = function(plr: Player, args: {string})
-				local clipper = Deps.Assets.Clipper:Clone()
-				clipper.Name = "ADONIS_NoClip"
-
 				for i, p in service.GetPlayers(plr, args[1]) do
 					Admin.RunCommand(Settings.Prefix.."clip", p.Name)
-					local new = clipper:Clone()
-					new.Parent = p.Character.Humanoid
-					new.Disabled = false
```
Comment (P3tray) [inline MainModule/Server/Commands/Moderators.lua:2516]: i dont mean to be rude but you've legit made all of this bit worse

File: MainModule/Server/Commands/Moderators.lua [hunk 2]
```diff
@@ -2538,17 +2539,12 @@ return function(Vargs, env)
...
 			Commands = {"clip", "unnoclip"};
 			Args = {"player"};
-			Description = "Un-NoClips the target player(s)";
+			Description = "Un-NoClips the target player(s).";
 			AdminLevel = "Moderators";
 			Function = function(plr: Player, args: {string})
 				for i, p in service.GetPlayers(plr, args[1]) do
 					local old = p.Character.Humanoid:FindFirstChild("ADONIS_NoClip")
 					if old then
-						local enabled = old:FindFirstChild("Enabled")
```
Comment (P3tray) [inline MainModule/Server/Commands/Moderators.lua:2547]: this bit is nice if it works

**Review by P3tray — CHANGES_REQUESTED** [review]

File: MainModule/Server/Commands/Moderators.lua (hunk 1 above)
Comment (EgizianoEG) [reply MainModule/Server/Commands/Moderators.lua:2516]: wdym? I don't see a purpose to have an unused clipper script?

Comment (P3tray) [reply MainModule/Server/Commands/Moderators.lua:2516]: image
you've done this

Comment (EgizianoEG) [reply MainModule/Server/Commands/Moderators.lua:2516]: ??!

File: MainModule/Server/Commands/Moderators.lua [hunk 3]
```diff
@@ -2503,17 +2503,18 @@ return function(Vargs, env)
 			Prefix = Settings.Prefix;
 			Commands = {"noclip"};
 			Args = {"player"};
-			Description = "NoClips the target player(s); allowing them to walk through walls";
+			Description = "NoClips the target player(s); allowing them to walk through walls.";
```
Comment (EasternBloxxer) [inline MainModule/Server/Commands/Moderators.lua:2506]: why the dot

File: MainModule/Server/Commands/Moderators.lua (hunk 1 above)
Comment (EasternBloxxer) [reply MainModule/Server/Commands/Moderators.lua:2516]: wtf

File: MainModule/Server/Commands/Moderators.lua (hunk 3 above)
Comment (EgizianoEG) [reply MainModule/Server/Commands/Moderators.lua:2506]: Punctuation 🤷‍♂️

File: MainModule/Server/Commands/Moderators.lua (hunk 1 above)
Comment (ccuser44) [reply MainModule/Server/Commands/Moderators.lua:2516]: You should move a lot of the stuff back in the loop and only preserve it's cloning behavior

**Pull request closed without merging**